src/containers/ResourcePage/ResourcePage.jsx

```diff
@@ -60,9 +60,6 @@ const ResourcePage = props => {
     }
   }
 
-  if (typeof window != 'undefined' && window.scrollY) {
-    window.scroll(0, 0);
-  }
```

Comment:
Jeg klarte ikke helt skjønne hvorfor vi hadde denne og den gikk i beina på scrollingen, så jeg fjernet den.
Om dere finner noe i testing så si i fra :smile: 
(Ser vi allerede scroller til toppen ved navigering i `App.tsx`)

Reply:
Ikke noe om issue, tror jeg, men var du som la det til i denne https://github.com/NDLANO/ndla-frontend/commit/ab7ed043a

Reply:
Må ha vore for å være sikker på at vi havner på topp ved visning av ressurs. Plasser å sjekke:
* Gå til ressurs fra søkeside når du har scrolla langt ned
* Gå til ressurs fra sub-sub-emne
* Gå til ny ressurs fra launchpad

Reply:
Ser ut til å funke etter kjappe stikkprøver.